Money_Counter_JB.py

```diff
@@ -0,0 +1,44 @@
+# This programme counts British coins based on their weight
+
+# Started with dictionary of UK coin weights
+
+coindict = {
+    "1": 3.56,
+    "2": 7.12,
+    "5": 3.25,
+    "10": 6.5,
+    "20": 5.0,
+    "50": 8.0
+    }
+
+while True:     # Put this eternal loop in so I don't have to keep restarting it
+
+    print()     # Blank lines for aesthetics
+
+    objective = input("\nWould you like to \n\n(a) weigh coins \nor \n(b) fill a coin bag?\n\n " ) # First choice: find the value of a batch of coins or fill a bag correctly?
+        
+    if objective == ("b"): # The fill a bag option
```

Comment:
Now you've set a variable, you can do

```suggestion
    if objective == fill_bag_option:
```